.circleci/config.yml

```diff
@@ -116,6 +118,13 @@ jobs:
         parallelism: 1
         steps:
             - checkout
+            - attach_workspace:
+                at:  ~/transformers/test_preparation
+            - run: |
+                if [ ! -s test_preparation/test_list.txt ]; then
+                    echo "No tests to run, exiting early!"
+                    circleci-agent step halt
+                fi
```

Comment:
So given this, the workflow will always be there, but it will sometimes last ~0 seconds, I suppose?

Reply:
11s to be exact (it spins up and does the checkout, which can't be done after attaching the folder).